pkg/converter/testdata/docker-compose-v2.yml

```diff
@@ -10,8 +10,7 @@ services:
     links:
       - db
   wordpress:
-   depends_on:
-     - db
```

Comment:
As seen on the busy_box service, services which are declared in `depends_on` and `links` getting also resolved. So to ensure that the dependencies gets set by the host:port search - I've removed this.